systest/ldbc/alpha.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.5"
+services:
+  alpha1:
+    image: dgraph/dgraph:local
+    working_dir: /data/alpha1
+    labels:
+      cluster: test
+    ports:
+      - "8080"
+      - "9080"
+    volumes:
+      - type: bind
+        source: $GOPATH/bin
+        target: /gobin
+        read_only: true
+      - type: bind
+        source: ./out/0/p
+        target: /posting
+        read_only: false
+    command: /gobin/dgraph ${COVERAGE_OUTPUT} alpha --my=alpha1:7080 --zero=zero1:5080 -p=/posting --logtostderr
+      -v=2 
+      --security "whitelist=10.0.0.0/8,172.16.0.0/12,192.168.0.0/16;"
```

Comment:
@anurags92 I may be a little late to the party here, but this whitelist can be shrunk. This may be too open.

Reply:
Thanks, I'll remove this on the main. I think I just copied over what we had in bulkload tests.